src/SFA.DAS.EmployerAccounts.Web/Controllers/SearchPensionRegulatorController.cs

```diff
@@ -1,304 +1,305 @@
+            {
+                viewModel.PayeRef = viewModel.PayeRef.Insert(3, "/");
+            }
+
+            viewModel.AddErrorsFromDictionary(errors);
+        }
+
+        private async Task<bool> CheckIfPayeSchemeAlreadyInUse(string empRef)
+        {
+            var schemeCheck = await _mediatr.SendAsync(new GetPayeSchemeInUseQuery { Empref = empRef });
+
+            return schemeCheck.PayeScheme != null;
+        }
+
+        private async Task SavePayeDetails(string aorn, string payeRef)
+        {
+            await _mediatr.SendAsync(new SavePayeRefData(new EmployerAccountPayeRefData
+            {
+                PayeReference = payeRef,
+                AORN = aorn
+            }));
+        }
+
+        private async Task SavePensionRegulatorOrganisationDataIfItHasAValidName(
+            PensionRegulatorDetailsViewModel viewModel, bool newSearch, bool multipleResults)
+        {
+            if (viewModel?.Name != null)
+            {
+                await _mediatr
+                    .SendAsync(new SaveOrganisationData
+                    (
+                        new EmployerAccountOrganisationData
+                        {
+                            OrganisationReferenceNumber = viewModel.ReferenceNumber.ToString(),
+                            OrganisationName = viewModel.Name,
+                            OrganisationType = viewModel.Type,
+                            OrganisationRegisteredAddress = viewModel.Address,
+                            OrganisationStatus = viewModel.Status ?? string.Empty,
+                            NewSearch = newSearch,
+                            PensionsRegulatorReturnedMultipleResults = multipleResults
+                        }
+                    ));
+            }
+        }
+
+        private async Task SavePensionRegulatorDataIfItHasAValidName(PensionRegulatorDetailsViewModel viewModel, bool newSearch, bool multipleResults, string aorn, string payeRef)
+        {
+            if (viewModel?.Name != null)
+            {
+                await _mediatr
+                    .SendAsync(new SaveOrganisationAndPayeData
+                    (
+                        new EmployerAccountOrganisationData
+                        {
+                            OrganisationReferenceNumber = viewModel.ReferenceNumber.ToString(),
+                            OrganisationName = viewModel.Name,
+                            OrganisationType = viewModel.Type,
+                            OrganisationRegisteredAddress = viewModel.Address,
+                            OrganisationStatus = viewModel.Status ?? string.Empty,
+                            NewSearch = newSearch,
+                            PensionsRegulatorReturnedMultipleResults = multipleResults
+                        },
+                        new EmployerAccountPayeRefData { AORN = aorn, PayeReference = payeRef }
+                    ));
+            }
+        }
+
+    }
 }
```

Comment:
what has happened to this file?

Reply:
Tidied up (just the and of the file, if I format the whole file then I'd be tidying up those looooooong paraaaamaaarter lines as well)